udata_gouvfr/settings.py

```diff
@@ -13,3 +13,8 @@
 # api.gouv.fr
 APIGOUVFR_URL = 'https://api.gouv.fr/api/v1/apis'
 APIGOUVFR_ALLOW_OPENNESS = ['open', 'semi_open']
+
+# Newsletter's subscription page's link
+NEWSLETTER_SUBSCRIPTION_BANNER = False
+NEWSLETTER_LINK = ''
```

Comment:
you can add those values as default https://github.com/etalab/udata-gouvfr/pull/512#issuecomment-733806815